docs/UserGuide.adoc

```diff
@@ -84,32 +87,245 @@ Examples:
+
+* `addexpense 1 detail/iPHONE11 amountOTHR/800` +
+Adds an expense incurred on the 1st day of the trip of an `iPHONE11` for $800. This amount will be converted to SGD using *exchange rate* provided.
+
+
+===== Editing an expense: `editexpense`
+
+Allow the user to edit an expense. The expense must be already added to the expense tracker.
+
+Format: `editexpense INDEX [detail/DETAIL] [amountSGD/AMOUNTSGD] [amountOTHR/AMOUNTOTHR]`
+
+****
+* Edits the expense at the specified `INDEX`. The index refers to the index number shown in the displayed expense list. The index *must be a positive integer* 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* An *exchange rate* is required if the user is entering the amount in another currency.
+* If the user enters the expense in other currency, the amount will be automatically converted to SGD using the *exchange rate* provided.
+****
+
+Examples:
+
+* `editexpense 1 detail/MacbookPro 13 amountSGD/3000` +
+Edit the details and amount paid in SGD of the 1st expense to be `MacbookPro 13` and `$3000 SGD` respectively.
+
+* `editexpense 1 detail/AirPods amountOTHR/300` +
+Edit the details and amount paid in other currency of the 1st expense to be `AirPods` and `$300` respectively. The amount will be automatically converted to SGD using *exchange rate* provided.
+
+===== Deleting an expense: `deleteexpense`
+
+Allow the user to delete an expense. The expense must be already added to the expense tracker.
+
+Format: `deleteexpense INDEX`
+
+Examples:
+*deleteexpense 1 +
+Delete the 1st item in the expense tracker.
+
+
+===== Listing down expenses for a day: `list`
+
+Allow the user to see their expense for a specific day of the trip as a list.
+
+Format: `list DAY`
+
+****
+* Shows a list of expense on a specified `DAY`. The day refers to a specific day of the trip. The day *must be a positive integer* and be *lesser than the total number of days of the trip*.
+* If `DAY` is entered as 0, all expenses incurred during the trip will be shown as a list.
 ****
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `list 1` +
+Shows the expenses incurred on the 1st day of the trip.
+
+* `list 0` +
+Shows all expenses incurred during the whole duration of the trip
+
+==== Recommend ways to save money based on prevailing usage [Coming in v2.0]
+
+Use complex algorithms to monitor and track the user's spending habits. Recommend ways to save money on certain items.
```

Comment:
Grammar: "**Uses** complex", "**Recommends** ways ..."

Both "user's spending habits" and "user spending habits" should be acceptable.